docs/index.md

```diff
@@ -0,0 +1,15 @@
+# Documentation
+
```

Comment:
Can you add a link to this somewhere in the README file? This is incredibly helpful and I don't want people to miss it.

Reply:
Doing.

Reply:
Done.